distributed/protocol/utils.py

```diff
@@ -92,7 +92,11 @@ def merge_frames(header, frames):
         if len(L) == 1:  # no work necessary
             out.extend(L)
```

Comment:
I'm interested to know why we don't run into this case. Are we seeing the frames get split? If so, should we be disabling that? It seems we should still merge them somehow if they are getting split, but if we can avoid the splitting that would be better still.

Reply:
frame splitting is part of regular serialization:
https://github.com/dask/distributed/blob/f2f82c6c2e8d36731cb3fb82fb1f80ea0323358e/distributed/protocol/core.py#L52

Reply:
+1 to @jakirkham 's comment

Before putting a band-aid on this problem I think that we should first see if we can avoid the situation entirely.